tests/test_strategies.py

```diff
@@ -76,9 +76,9 @@ def run_many(self, positions):
         return [self.fake_priors] * len(positions), [self.fake_value] * len(positions)
 
 
-def initialize_basic_player():
+def initialize_basic_player(position=None):
```

Comment:
(i guess i'm saying i think the behavior should be
```
if position is not None:
    [do search]
```
)

Reply:
You do the search so that you have priors and a subsequent tree_search actually does something interesting (instead of just returning the root node x 8).

Reply:
I agree that makes sense, and we don't seem to manually push in priors.  maybe just call it init_player_with_search? or something?